charts/argo-workflows/templates/controller/workflow-controller-config-map.yaml

```diff
@@ -97,10 +92,22 @@ data:
     {{- end}}
     {{- if .Values.controller.metricsConfig.enabled }}
     metricsConfig:
-{{ toYaml .Values.controller.metricsConfig | indent 6}}{{- end }}
+      enabled: {{ .Values.controller.metricsConfig.enabled }}
+      path: {{ .Values.controller.metricsConfig.path }}
+      port: {{ .Values.controller.metricsConfig.port }}
+      metricsTTL: {{ .Values.controller.metricsConfig.metricsTTL }}
+      ignoreErrors: {{ .Values.controller.metricsConfig.ignoreErrors }}
+      secure: {{ .Values.controller.metricsConfig.secure }}
```

Comment:
I'm not so sure but it looks that we cannot pass unsupported values on argo-workflow-configmap... 🤔

Reply:
I think that has to do with [this](https://argoproj.github.io/argo-workflows/upgrading/#feat-fail-on-invalid-config-8295).  right?

Reply:
The upstream [PR](https://github.com/argoproj/argo-workflows/pull/8295)

Reply:
Oh that's the one... 😇  Thank you!